### ISSUE TITLE
[$50] Android - Workspace - The Next button overlaps with the device navigation bar

### ISSUE THREAD

**Issue by IuliiaHerets** [Expensify/App#54637]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** 9.0.79-0
**Reproducible in staging?:** Yes
**Reproducible in production?:** Yes
**If this was caught on HybridApp, is this reproducible on New Expensify Standalone?:** Yes, reproducible on both
**If this was caught during regression testing, add the test name, ID and link from TestRail:** https://expensify.testrail.io/index.php?/tests/view/5383034
**Issue reported by:** Applause Internal Team
**Device used:** Samsung S23FE/ Android 14
**App Component:** Workspace Settings

## Action Performed:
Preconditions:
The user is an admin of a workspace.
The device navigation bar is displayed instead of gestures.

Steps:
1. Navigate to More features on workspace settings
2. Enable Company Cards
3. Navigate to Company Cards
4. Tap on Add Company Cards
5. Tap on American Express
6. Tap on the next button
7. Select the option for "Corporate Cards"
8. Tap on the next button

## Expected Result:
The "Next" button should be displayed above the device navigation bar.

## Actual Result:
The "Next" button is displayed behind the device navigation bar.

## Workaround:
Unknown

## Platforms:
- [x] Android: Standalone
- [x] Android: HybridApp
- [ ] Android: mWeb Chrome
- [ ] iOS: Standalone
- [ ] iOS: HybridApp
- [ ] iOS: mWeb Safari
- [ ] MacOS: Chrome / Safari
- [ ] MacOS: Desktop

## Screenshots/Videos
![Bug6703252_1735351198936!next_button_1](https://github.com/user-attachments/assets/aaf109ac-13be-4607-9bfe-09d30fd61849)
![Bug6703252_1735351198937!next_button_2](https://github.com/user-attachments/assets/fa96a441-c58c-4219-8c4c-bd25bad840a5)

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)
<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~021873112388502156931</li>
        <li>Upwork Job ID: 1873112388502156931</li>
        <li>Last Price Increase: 2025-01-04</li>
    </ul>
</details>

<details><summary>Issue Owner</summary>Current Issue Owner: @abdulrahuman5196</details>

**Comment by BhuvaneshPatil** [comment]
## Proposal

### Please re-state the problem that we are trying to solve in this issue.
while adding company card, on card instruction page, we have next button going under device navigation bar

### What is the root cause of that problem?
We are applying `safeAreaPaddingBottomStyle` to the `ScrollView` component in `CardInstructionsStep`.
Thats why it takes full height and as button is at end, it goes behind navigation bar.

### What changes do you think we should make in order to solve the problem?
We pass the `safeAreaPaddingBottomStyle` in `contentContainerStyle` array to `ScrollView` like following - 

```tsx
 <ScreenWrapper
            testID={CardInstructionsStep.displayName}
            includeSafeAreaPaddingBottom={false}
            shouldEnablePickerAvoiding={false}
            shouldEnableMaxHeight
        >
            {({ safeAreaPaddingBottomStyle }) => (
                <>
                    ...
                    <ScrollView
                        style={[styles.pt0, styles.trialBannerBackgroundColor]}
                        contentContainerStyle={[safeAreaPaddingBottomStyle, styles.flexGrow1]}
                    >
                     ...
                </>
            )}

        </ScreenWrapper>
```
 
We have followed same pattern at multiple places and it works as expected.

Example - https://github.com/Expensify/App/blob/9412d2147b4b44b4371f8bab19c288db5b9bd23a/src/pages/FlagCommentPage.tsx#L133-L146

### What specific scenarios should we cover in automated tests to prevent reintroducing this issue in the future?


### What alternative solutions did you explore? (Optional)

**Comment by huult** [comment]
## Proposal

### Please re-state the problem that we are trying to solve in this issue.

The Next button overlaps with the device navigation bar

### What is the root cause of that problem?

This occurs because we have disabled `includeSafeAreaPaddingBottom` by setting it to false here.

https://github.com/Expensify/App/blob/1ab06c4c6f838aac232efc5883cd309efe01fac7/src/pages/workspace/companyCards/addNew/CardInstructionsStep.tsx#L88

### What changes do you think we should make in order to solve the problem?


To resolve this issue we just need remove `includeSafeAreaPaddingBottom` [here](https://github.com/Expensify/App/blob/1ab06c4c6f838aac232efc5883cd309efe01fac7/src/pages/workspace/companyCards/addNew/CardInstructionsStep.tsx#L88)

### What specific scenarios should we cover in automated tests to prevent reintroducing this issue in the future?


I don’t think we need a test here since this is a UI bug

### What alternative solutions did you explore? (Optional)

**Reminder:** Please use plain English, be brief and avoid jargon. Feel free to use images, charts or pseudo-code if necessary. Do not post large multi-line diffs or write walls of text. Do not create PRs unless you have been hired for this job.

**Comment by mountiny** [comment]
making this external for $50 as its very minor 

@abdulrahuman5196 can you please review the proposals

**Comment by kirillzyusko** [comment]
I like the proposal from @huult - this is the current appraoch in the app (i. e. we don't subscribe to context via inline functions and it's something that we actively use).

The only one thing that concerns me is that the button (Next) seems to be very high (at least for me). Just make sure it has the same position as on other screens and we're good to go!

**Comment by huult** [comment]
@kirillzyusko Yes, I will test it on other screens to ensure the same position

**Comment by abdulrahuman5196** [comment]
@mountiny I think we can handle similar issues here https://github.com/Expensify/App/issues/53650 instead of handling it separately.

**Comment by abdulrahuman5196** [comment]
planning to handle it here https://github.com/Expensify/App/issues/53650
We can close this @mountiny
